frontend/components/commentView.tsx

```diff
@@ -1,9 +1,36 @@
 // import Image from 'next/image'
 import AnonPill, { NounSet } from "./anonPill";
+import * as React from "react";
+import dayjs from "dayjs";
+const relativeTime = require("dayjs/plugin/relativeTime");
+dayjs.extend(relativeTime);
 
-interface ICommentViewProps {}
+interface ICommentViewProps {
+  message: string;
+  groupType: number;
+  createdAt: Date;
+  proof: string;
+}
+
+const CommentView: React.FC<ICommentViewProps> = ({
+  message,
+  proof,
```

Comment:
@salvinoarmati did not put this in the UI cause we didn't talk about what the design should look like but this is ready for u to put somewhere here once we finalize the design!